### DISCUSSION TITLE
Curso.

### DISCUSSION THREAD

**Discussion by patyberger** [community/community#127203] [Programming Help]
### Body

Bom dia, estou com dificuldades para iniciar o curso, totalmente perdida. Preciso de ajuda. Obrigada. Patricia

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by GuilhermeNobrega** [comment]
Bom dia!

Para iniciar sua jornada na área de tecnologia, acredito que é essencial focar em três pilares fundamentais de estudo:

1) Inglês:
Estude e pratique muito inglês. O domínio da língua inglesa abre muitas portas no mercado de trabalho, tanto no Brasil quanto internacionalmente. Muitos recursos, documentação técnica e comunidades de desenvolvedores utilizam o inglês como língua principal, facilitando o acesso a conhecimento e oportunidades.

2) Pratique Lógica de Programação:
Escolha uma linguagem de programação para iniciar seus estudos. Eu comecei com Python, mas, em retrospectiva, recomendo fortemente começar com C ou Java. Essas linguagens são robustas para desenvolver uma lógica de programação sólida e entender tópicos importantes como gerenciamento de memória, estrutura de dados e algoritmos. Após ter uma boa base, é importante praticar criando pequenos ou grandes projetos (como uma calculadora, conversor de binário, etc.). Python continua sendo uma excelente linguagem para criar soluções inovadoras e vale a pena ser aprendida depois.

3) Aprenda Sistema Operacional Linux e Git:
Um profissional de TI que não tem noção de sistemas operacionais Linux é um profissional incompleto. A importância de aprender Linux se dá pela possibilidade de futuramente você trabalhar com servidores Linux, máquinas Linux, entre outros. Além disso, entender o funcionamento de sistemas operacionais melhora seu conhecimento técnico geral. Git é outra ferramenta essencial para qualquer desenvolvedor, pois permite o versionamento de código e facilita a colaboração em projetos. Aprender a usar Git e criar um portfólio no GitHub é crucial, pois você mostra para futuros avaliadores os projetos que você tem em sua nuvem.

Sugestões Adicionais:
Matemática e Algoritmos: Aprofunde-se em matemática e algoritmos. Muitos problemas de programação exigem uma compreensão sólida desses conceitos.

Redes de Computadores: Compreender os fundamentos de redes de computadores é vital para entender como os dados se movem na internet e como as aplicações se comunicam.

Participação em Comunidades e Hackathons: Participar de comunidades de desenvolvedores e hackathons pode acelerar seu aprendizado, proporcionar networking e abrir portas para oportunidades de carreira.

Citei esses tópicos pois comecei por eles e grandes profissionais com que compartilho ambiente de trabalho também recomendam esse caminho de estudos.. lhe desejo boa sorte e boas vindas ao mundo da T.I!

**Comment by patyberger** [comment]
Perfeito, mas me refiro ao curso exatamente, onde localizar as aulas.
No primeiro acesso consegui ver aula inicial, onde mostrava todo o
ambiente, mas agora não consigo localizar.

Em ter., 4 de jun. de 2024 às 08:44, Wilhelm ***@***.***>
escreveu:

**Reply by GuilhermeNobrega** [reply]
Que curso você está falando? curso do github?

**Comment by patyberger** [comment]
Sim, isso mesmo. Me inscrevi no  Curso de Ciência da Computação de Harvard
2022, mas não consigo encontrar as aulas.

Em ter., 4 de jun. de 2024 às 08:53, Wilhelm ***@***.***>
escreveu:

**Comment by GuilhermeNobrega** [comment]
Você se inscreveu aqui?

https://www.estudarfora.org.br/cursos/cc50/

O curso do github é diferente deste. Esse é para iniciantes da tecnologia, mas não ensina muita coisa.. você está começando agora, certo?

**Comment by patyberger** [comment]
Sim, isso mesmo.
Não consigo mais acessar as aulas.
Pode me ajudar?

Em ter., 4 de jun. de 2024 às 09:06, Wilhelm ***@***.***>
escreveu:

**Comment by ebndev** [comment]
Hi @patyberger, thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting to ensure discussions are adhering to our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md). 

We’ll be locking this post for now, but please feel encouraged to create a new discussion in English if you still need support.
